packages/webdriver/src/utils.js

```diff
@@ -286,14 +286,10 @@ export function isAndroid (caps) {
  */
 export function isSauce (hostname, caps) {
     return Boolean(
-        hostname &&
```

Comment:
This will not work for people who are still using Sauce without W3C because not all users are using `extendedDebugging`

Reply:
@wswebcreation the flag only indicates if the custom Sauce commands can be applied to the client. It was doing this before too. I agree with you though that this can be confusing for people that don't run extended debugging jobs and want to use the flag. I would like to tackle this in a different PR thought to clean this flag up and have two flags for running on Sauce and for running on Sauce with extended debugging.

Reply:
https://github.com/webdriverio/webdriverio/issues/4196